ktxspec.adoc

```diff
@@ -1134,6 +1140,34 @@ The bitstream of the UASTC data is described in Chapter 25
 {url-df-spec}#UASTC[_UASTC Compressed Texture Image Format_] of
 <<KDF14>>.
 
+Basis Universal ASTC HDR Format::
+The Universal ASTC HDR image format (UASTC HDR) is indicated by
+`colorModel` `KHR_DF_MODEL_UASTC_HDR` (= 167). This format supports
+two compression ratios defined by the texel block size in the DFD:
+4x4 and 6x6. Images in this format are a strict RGB-only ASTC HDR
+subset so `vkFormat` must be set to whichever of
+`VK_FORMAT_ASTC_4x4_SFLOAT_BLOCK` (= 1000066000) or
+`VK_FORMAT_ASTC_6x6_SFLOAT_BLOCK` (= 1000066004) matches the texel
+block size. The DFD must be as described in Section ??? of <<KDF14>>.
+Images in this format must be transcoded to BC6H or decoded to raw
+16-bit floating-point values before being uploaded to and sampled
+by a GPU that does not support ASTC HDR.  UASTC HDR images can be
+supercompressed with any scheme except BasisLZ (`supercompressionScheme`
+= 1) and Basis GPU Photo 6x6 (`supercompressionScheme` = 4). If
```

Comment:
If I am understanding correctly the ASTC 6x6 part of "UASTC HDR 6x6 Intermediate" is the same as UASTC HDR 6x6 as @richgel999 has stated the same encoder is used. Therefore what I would like to say here is

"UASTC HDR 4x4 images can be supercompressed with any scheme except BasisLZ and Basis GPU Photo 6x6. UASTC 6x6 images can be supercompressed with any scheme except BasisLZ. Best results are obtained with Basis GPU Photo 6x6."

@richgel999 is this okay.

Reply:
It sounds like we do need a "UASTC HDR 6x6 Intermediate" section separate from the "UASTC HDR" section.